shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -331,6 +331,14 @@ export default {
 
       const out = findBy(this.versionOptions, 'value', str);
 
+      // Adding the option 'none' to Container Network select (used in Basics component)
+      // https://github.com/rancher/dashboard/issues/10338
+      // there's an update loop on refresh that might include 'none'
+      // multiple times... Prevent that
+      if (out.serverArgs.cni?.options && !out.serverArgs.cni.options.includes('none')) {
```

Comment:
I'd probably do 
if (out?.serverArgs?.cni?.options && !out.serverArgs.cni.options.includes('none')) {
just in case